x-pack/plugins/security_solution/cypress/integration/timelines/creation.spec.ts

```diff
@@ -95,3 +99,28 @@ describe('Timelines', (): void => {
     });
   });
 });
+
+describe('Create a timeline from a template', () => {
+  before(() => {
+    cleanKibana();
+    loginAndWaitForPageWithoutDateRange(TIMELINE_TEMPLATES_URL);
+    waitForTimelinesPanelToBeLoaded();
+  });
+
+  it('Should have the same query', () => {
+    createTimelineTemplate(timeline);
+
+    cy.get(TIMELINE_COLLAPSED_ITEMS_BTN).first().click();
+    cy.intercept('PATCH', '/api/timeline').as('timeline');
+
+    cy.get(TIMELINE_CREATE_TIMELINE_FROM_TEMPLATE_BTN).click({ force: true });
+
+    cy.wait('@timeline').then(({ request }) => {
+      expect(request.body.timeline).to.haveOwnProperty('description', timeline.description);
+      expect(request.body.timeline.kqlQuery.filterQuery.kuery).to.haveOwnProperty(
```

Comment:
Is there any reason why we are asserting the BE responses instead of checking on the UI?

Reply:
I'll check the UI and make sure timeline's modal appears.